invenio_oaiserver/query.py

```diff
@@ -45,6 +47,24 @@ def match(self, record):
         return self.query.accept(MatchUnit(record))
 
 
+def get_affected_records(spec=None, search_pattern=None):
+    """Get list of affected records."""
+    sets = '_oai.sets:{0}'.format(spec) if spec else None
```

Comment:
Shouldn't the double quotes be around the spec - `'_oai.sets:"{0}"'.format(spec)`?